src/GlobalPayments.Api/Terminals/ConnectionConfig.cs

```diff
@@ -47,6 +51,7 @@ public interface ITerminalConfiguration {
         Parity Parity { get; set; }
         StopBits StopBits { get; set; }
         DataBits DataBits { get; set; }
+        Handshake Handshake { get; set; }

```

Comment:
What is this used for?

Reply:
Handshake is used for terminal communication through serial port.